pkgs/tools/filesystems/bees/bees-service-wrapper

```diff
@@ -0,0 +1,196 @@
+#!@bash@/bin/bash
+PATH=@coreutils@/bin:@utillinux@/bin:@btrfsProgs@/bin:$PATH
+beesd_bin=@bees@/lib/bees/bees
+# PLEASE KEEP NIX-ISMS ABOVE THIS LINE TO EASE UPSTREAM MERGE
+#!/usr/bin/env bash
+
+# Upstream wrapper requires UUID to be used for configuration.
+
+# However, when declaratively describing a host, we may not know its UUID, and
+# shouldn't need to persist something that will differ between hosts built from
+# the same configuration template.
+
+# Thus, for using bees from NixOS, we have our own wrapper, which supports not
+# just UUID but any specification permitted by findmnt
+
+[[ $bees_debug ]] && { PS4=':${BASH_SOURCE##*/}:$LINENO+'; set -x; }
+
+usage() {
+  cat >&2 <<EOF
+Usage: ${BASH_SOURCE##*/} run|cleanup config-name|fs-spec [idxSizeMB=...] [verbosity=...] [workDir=...] [-- options...]
```

Comment:
Maybe it is a good idea to say explicitly that `options` are the daemon options (implying they are ignored by the script)?

Reply:
Changed the name to `daemon-options`, and also added some more explanatory text later in the heredoc.